src/bin/julialauncher.rs

```diff
@@ -219,8 +217,8 @@ fn run_app() -> Result<i32> {
     if args.len() > 1 {
         let first_arg = &args[1];
 
-        if first_arg.starts_with("+") {
-            julia_channel_to_use = Some(first_arg[1..].to_string());
+        if let Some(stripped) = first_arg.strip_prefix('+') {
+            julia_channel_to_use = Some(stripped.to_string());
```

Comment:
This way, we avoid manually accessing strings, which can throw errors like in Julia, and also only do the string stripping once.

Reply:
Oh, that is nice!